lib/internal/streams/operators.js

```diff
@@ -186,15 +186,15 @@ function asIndexedPairs(options = undefined) {
   }.call(this);
 }
 
-async function some(fn, options) {
+async function some(fn, options = {}) {
```

Comment:
We don't need to create a throw away object.
```suggestion
async function some(fn, options = undefined) {
```